### ISSUE TITLE
whitelist entry by single domain

### ISSUE THREAD

**Issue by idnovic** [nextdns/lists#4]
Please add a feature to allow me to give you a whole whitelist instead of me adding urls one by one. I want to link a github txt as whitelist because you can not expect a user the whitelist many domains by hand. Or let me import a whitelist/blacklist in the interface.

**Comment by romaincointepas** [comment]
Which whitelist(s) would you like to add?

Also, did you encounter any case where a website/app was broken because of a blocked domain and wouldn't have been if you used the above whitelists?

**Comment by idnovic** [comment]
as example I whitelisted:
ocsp.apple.com because I find oscp checks usefull
dns.google.com because I remeber some hardware/software to only check with google dns for connection

The problem is the more privacy friendly block lists block to much but they are correct 95% of the time.
See kowabit.de - bl*cklist of death because thename is fitting, blocks to much.
So I think a good whitelist is essiantial.

If I would add a whitelist now I would use:
https://github.com/Ultimate-Hosts-Blacklist/whitelist/blob/master/domains.list
https://raw.githubusercontent.com/raghavdua1995/DNSlock-PiHole-whitelist/master/whitelist.list
https://github.com/AdAway/AdAway/wiki/ProblematicApps

But I also like to be able to export my whitelist/blacklist.

**Comment by romaincointepas** [comment]
We will be adding a "do not block affiliate/tracking links" option very soon, which I think would cover most cases of users needing to import a whitelist (at least that's the feeling we are getting).

Regarding those 3 whitelists, we checked how many of those domains were actually blocked by lists and we found out that aside from affiliate/tracking, it's a small minority. Most of them were blocked at some point and are now whitelisted by all popular lists out there.

We may add the ability to import a file to your Whitelist/Blacklist in the future, we will see the % of users still needing this after the affiliate/tracking option.

Side note: highly recommending our subreddit [/r/nextdns](https://www.reddit.com/r/nextdns) for feature requests or UI/UX suggestions as more people can jump in to discuss it.
